app/models.py

```diff
@@ -240,6 +240,10 @@ class EmailBranding(db.Model):
     brand_type = db.Column(
         db.String(255), db.ForeignKey("branding_type.name"), index=True, nullable=False, default=BRANDING_ORG
     )
+    created_at = db.Column(db.DateTime, nullable=True, default=lambda: datetime.datetime.utcnow())
+    created_by = db.Column(UUID(as_uuid=True), db.ForeignKey("users.id"), nullable=True)
+    updated_at = db.Column(db.DateTime, nullable=True, onupdate=lambda: datetime.datetime.utcnow())
```

Comment:
Optional, but we use a slightly different format for `default` and `onupdate` model values elsewhere in this file - we just pass the function name as an argument without needing a lambda (`default=datetime.datetime.utcnow`  or `onupdate=datetime.datetime.utcnow`). We could stick to that for consistency or to avoid confusion, but both ways of doing things work.